generators/client/templates/angular/src/main/webapp/_index.html

```diff
@@ -41,6 +43,11 @@
             script.type = 'text/javascript';
             document.getElementsByTagName('head')[0].appendChild(script)
         }
+        /*if ('serviceWorker' in navigator) {
```

Comment:
This is going to create "code smells" in Sonar (we already have the issue for the Google Analytics code which is commented), could this be done another way?
Otherwise it's not really important, once minified this won't make any difference